generators/tnoc

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+import os
+import sys
+import glob
+
+templ = """/*
+:name: TnoC
+:description: Full TNoC core test
+:files: {0}
+:tags: tnoc
```

Comment:
this tag should correspond to the entry in `lrm.conf` in which you used different casing (`TNoC`)

Reply:
I've fixed